README.md

```diff
@@ -24,7 +24,7 @@ server.register([
 ```
 
 #### Options
-All options `(defaultRate, redisClient, overLimitError)` are required for the plugin to work properly.
+All options `(defaultRate, requestAPIKey, redisClient, overLimitError)` are required for the plugin to work properly.
```

Comment:
just `apiKey` might be misleading as to refer to the actual value of the key itself, so thought `requestAPIKey` would be better since the function takes in a `Request` object